spec/models/archival_media_collection_feature_spec.rb

```diff
@@ -43,7 +43,7 @@
 
       expect(page).to have_css ".btn-bag-path.browse-everything"
       click_on(class: "btn-bag-path")
-      expect(page).to have_css "#browse_everything"
```

Comment:
it still has this id?

Reply:
Yeah I didn't spend a lot of effort moving CSS around because this feature in general is intended to go away in #5409.